simple_network_sim/network_of_populations.py

```diff
@@ -83,10 +100,26 @@ def basicSimulationInternalAgeStructure(network, timeHorizon):
 
 # CurrentlyInUse
 def totalIndividuals(nodeState):
+    """Calculate the total population size over all ages and regions.
+
+    :param nodeState: The disease status of the population stratified by age.
+    :type nodeState: A dictionary with a tuple of (age, state) as keys and the number of individuals
+    in that state as values.
+    :return: The total number of individuals.
+    :rtype: float
+    """
     return sum(nodeState.values())
 
 
 def getAges(node):
+    """Get the unique set of ages from the node.
```

Comment:
A set is already unique

Reply:
This was sloppy of me; I meant "unique collection of" but set is a better word.